openlibrary/templates/home/index.html

```diff
@@ -10,22 +10,43 @@
     or <a href="/search?subject_facet=Accessible%20book">search for accessible books</a>.
 </div>
 
-$:render_template("home/splash")
-
-<div id="contentBody">
+<div class="page-banner page-banner-home sansserief">
```

Comment:
Misspelling: "sansserief" should be "sansserif" (though better would be "sans-serif").